__tests__/app.test.js

```diff
@@ -77,3 +77,37 @@ describe('GET /api/reviews/:review_id', () => {
 
 });
 
+describe('GET /api/reviews', () => {
+  test('Get status 200 response', () => {
```

Comment:
I'd argue this test isn't needed as the next 2 tests prove we're getting a status 200 code back anyway